StorageManagementSystem/Agent/StageRequestAgent.py

```diff
@@ -222,23 +216,23 @@ def _getOfflineReplicas( self ):
   def __usage( self, storageElement ):
     """ Retrieve current usage of SE
     """
-    if not storageElement in self.storageElementUsage:
-      self.storageElementUsage[storageElement] = {'TotalSize': 0.}
+    # Set it if not yet done
+    self.storageElementUsage.setdefault( storageElement, {'TotalSize': 0.} )
     return self.storageElementUsage[storageElement]['TotalSize']
 
   def __cache( self, storageElement ):
     """ Retrieve cache size for SE
     """
-    if not storageElement in self.storageElementCache:
-      self.storageElementCache[storageElement] = gConfig.getValue( "/Resources/StorageElements/%s/DiskCacheTB" % storageElement, 1. ) * 1000. / THROTTLING_STEPS
+    self.storageElementCache.setdefault( storageElement,
```

Comment:
remember that when calling setdfefault, the second argument is always evaluated, even if not needed. So it is maybe an overkill here ?